libr/crypto/crypto.c

```diff
@@ -36,6 +47,16 @@ R_API const char *r_crypto_name(ut64 bit) {
 	return "";
 }
 
+R_API const char *r_crypto_encoder_name(ut64 bit) {
```

Comment:
i would do a typedef for this ` typedef ut64 RCryptoEncoder` instead of passing ut64 here. also make it const

Reply:
what about `RCryptoSelector` or `RCryptoBit`, so it can be used for `r_crypto_name(...)` too ?